### ISSUE TITLE
hlx installation via curl occasionally fails due to rate limit

### ISSUE THREAD

**Issue by stefan-guggisberg** [adobe/helix-cli#400]
**Description**
`hlx` installation via `curl` occasionally fails due to Github API rate limits.

**To Reproduce**
Check current rate limit for unauthenticated API calls:
```
curl https://api.github.com/rate_limit
```
If `resources.core.remaining == 0` run:
```
curl -sL https://www.project-helix.io/cli.sh | sh
```
or 
```
curl -sL https://raw.githubusercontent.com/adobe/helix-cli/master/cli_install.sh | sh
```

**Expected behavior**
`hlx` installation via `curl` should not be affected by Github API rate limits.

**Comment by kptdobe** [comment]
While do you need the `if` and cannot use the script on project-helix.io without checking the rate limit... in all the cases ? ;)

I hit the limit too this morning while developing. You might pass the installation and right after hit the limit. Same result.

**Comment by kptdobe** [comment]
I think we should really tackle https://github.com/adobe/helix-cli/issues/380 instead

**Comment by stefan-guggisberg** [comment]
> While do you need the `if` and cannot use the script on project-helix.io without checking the rate limit... in all the cases ? ;)

sorry, i can't follow you here.

**Comment by kptdobe** [comment]
Oh sorry, I misread. Forget about this part ;)

**Comment by stefan-guggisberg** [comment]
fixed in 41d221f
